Assets/MixedRealityToolkit/Common/Scripts/CrossPlatform/ReflectionExtensions.cs

```diff
@@ -1,10 +1,9 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-namespace MixedRealityToolkit.Common.Extensions
```

Comment:
Why has this namespace changed, as it is located in the "Common\Extensions" folder?
Plus it dirties the base MRTK namespace

Reply:
These extensions implement some reflection methods that don't exist in UWP but do exist elsewhere. Since the SWIG generated files use these methods, putting this file into a different namespace would require manually adding a using statement every time they're regenerated. In the interest of cutting down on manual steps and preventing the need to edit generated files, these would have to be somewhere visible to the SWIG files.

These two files are more similar to `SystemException.cs` and `MetroHandleRef.cs` than the other extension scripts, since they're redefining things that exist elsewhere. I believe these two files used to live without any namespace.

Reply:
My point there would be (as per my comment below) then we should look to fix the generator for the future, rather than muddying / dirtying the API with rogue classes in the base namespace.
Fixing the source rather than the symptom.